ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -253,7 +254,7 @@ protected void submitResponse(RestRequest restRequest, RestResponseChannel restR
             RestServiceErrorCode.getRestServiceErrorCode(((RouterException) exception).getErrorCode()));
       }
       responseHandler.handleResponse(restRequest, restResponseChannel, responseBody, exception);
-    } catch (RestServiceException e) {
+    } catch (Exception e) {
```

Comment:
why Exception. Looks like only RestServiceException is thrown within the try block